docs/examples/variability-index.py

```diff
@@ -0,0 +1,69 @@
+"""
+Calculate Variability Index
+===========================
+
+Calculate the Variability Index for a GHI time series.
+"""
+
+# %%
+# Highly variable irradiance can cause mismatch between irradiance and
+# power measurements and result in noisy performance metrics. As such,
+# identifying and removing highly variable conditions is useful in
+# certain analyses. Identification and quantification of highly variable
+# conditions are also of interest in grid integration and hourly modeling
+# error contexts. The variability index (VI) is one way of quantifying the
+# variability or jaggedness of an irradiance signal relative to a
+# corresponding reference clear-sky irradiance profile. Note that
+# quantifying variability is related to but distinct from clear-sky
+# detection. For example, both clear and overcast skies have low VI. This
+# example uses GHI data collected from the NREL RMIS system to calculate
+# the variability index as a time series.
+
+import pvanalytics
+from pvanalytics.metrics import variability_index
+import matplotlib.pyplot as plt
+import pandas as pd
+import pathlib
+import pvlib
+
+# %%
+# First, read in data from the RMIS NREL system. This data set contains
+# 5-minute right-aligned POA, GHI, DNI, DHI, and GNI measurements,
+# but only the GHI is relevant here.
+
+pvanalytics_dir = pathlib.Path(pvanalytics.__file__).parent
+rmis_file = pvanalytics_dir / 'data' / 'irradiance_RMIS_NREL.csv'
+data = pd.read_csv(rmis_file, index_col=0, parse_dates=True)
+freq = '5T'
```

Comment:
This appears to be unused.

Reply:
In case anyone else is surprised like I was that the linter didn't complain about this with `F841`, I think it's because it doesn't know the difference between a script and a module that might be imported by someone else, and it's legitimate to define module-level variables that never get used in the module itself.  If this unused definition was inside a function then it would be flagged as expected.  Or put another way, `F841` is only for local variables, not globals.

Reply:
I removed the freq line in commit https://github.com/pvlib/pvanalytics/pull/153/commits/235d4dcd8ebeeb11ee401f9c3467b965d6e4622c